platform/linux-generic/pktio/mdev.c

```diff
@@ -0,0 +1,585 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include "config.h"
+
+#ifdef ODP_MDEV
+
+#include <odp_posix_extensions.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <linux/vfio.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <dirent.h>
+
+#include <config.h>
+#include <odp_posix_extensions.h>
```

Comment:
Duplicate `#include`? This is included above on line 11.

Reply:
Thanks, removed 2 duplicated headers.